### PR TITLE
Asset Delete

### PR DESCRIPTION
**Pull request by vivianliu0**
Loom Demo: https://www.loom.com/share/db1840fc26c34b909eef21d324c1425f

### CODE REVIEW

File: src/Components/Admin/admin.js [hunk 1]
```diff
@@ -279,6 +287,25 @@ export default function Admin() {
...
+
+    const handleDeleteAssetCancel = () => {
+        setDeleteAssetId(null);
+        setDeleteAssetModalOpen(false);
+    };
+
+    const handleDeleteAssetSubmit = async () => {
+        await deleteAsset(deleteAssetId, handleError);
+
+        const modifiedAssets = assets.filter((env) => env.id !== deleteAssetId);
```
Comment (ahmedhamodi) [inline src/Components/Admin/admin.js:303]: Nit: the iterated object variable name in the filter shouldn't be `env`. Can we do `asset` instead?

**Review by ahmedhamodi — COMMENTED** [review]
Code looks good. One minor nit

File: src/Components/Admin/admin.js (hunk 1 above)
Comment (vivianliu0) [reply src/Components/Admin/admin.js]: Thanks, updated :)

**Review by ahmedhamodi — APPROVED** [review]
Tested locally, works as expected. Good work 🎉

**Pull request merged**